src/client/java/teammates/client/scripts/sql/VerifyCourseEntityAttributes.java

```diff
@@ -160,6 +171,173 @@ private boolean verifyStudent(CourseStudent oldStudent,
+                && newSession.getCreatorEmail().equals(oldSession.getCreatorEmail())
+                && newSession.getInstructions().equals(SanitizationHelper.sanitizeForRichText(oldSession.getInstructions()))
+                && newSession.getStartTime().equals(oldSession.getStartTime())
+                && newSession.getEndTime().equals(oldSession.getEndTime())
+                && newSession.getSessionVisibleFromTime().equals(oldSession.getSessionVisibleFromTime())
+                && newSession.getResultsVisibleFromTime().equals(oldSession.getResultsVisibleFromTime())
+                && newSession.getGracePeriod().equals(Duration.ofMinutes(oldSession.getGracePeriod()))
+                && newSession.isOpeningEmailEnabled() == oldSession.isOpeningEmailEnabled()
+                && newSession.isClosingEmailEnabled() == oldSession.isClosingEmailEnabled()
+                && newSession.isOpenEmailSent() == oldSession.isSentOpenEmail()
+                && newSession.isOpeningSoonEmailSent() == oldSession.isSentOpeningSoonEmail()
+                && newSession.isClosedEmailSent() == oldSession.isSentClosedEmail()
+                && newSession.isClosingSoonEmailSent() == oldSession.isSentClosingEmail()
+                && newSession.isPublishedEmailSent() == oldSession.isSentPublishedEmail()
+                && newSession.getCreatedAt().equals(oldSession.getCreatedTime())
+                && (newSession.getDeletedAt() == oldSession.getDeletedTime()
+                        || newSession.getDeletedAt().equals(oldSession.getDeletedTime()));
+        if (!doFieldsMatch) {
+            log(String.format("Mismatched fields for session: %s, course id: %s",
+                    oldSession.getFeedbackSessionName(), oldSession.getCourseId()));
+            return false;
+        }
+
+        List<teammates.storage.sqlentity.FeedbackQuestion> newQuestions = newSession.getFeedbackQuestions();
+        List<FeedbackQuestion> oldQuestions = ofy().load().type(FeedbackQuestion.class)
+                .filter("courseId", newSession.getCourse().getId())
+                .filter("feedbackSessionName", newSession.getName()).list();
+
+        if (newQuestions.size() != oldQuestions.size()) {
+            log(String.format("Mismatched question counts for session: %s, course id: %s",
+                    oldSession.getFeedbackSessionName(), oldSession.getCourseId()));
+            return false;
+        }
+
+        Map<Integer, FeedbackQuestion> questionNumberToOldQuestionMap = oldQuestions.stream()
+                .collect(Collectors.toMap(FeedbackQuestion::getQuestionNumber, question -> question));
+
+        return newQuestions.stream().allMatch(newQuestion -> {
+            FeedbackQuestion oldQuestion = questionNumberToOldQuestionMap.get(newQuestion.getQuestionNumber());
+            return verifyFeedbackQuestion(oldQuestion, newQuestion);
+        });
+    }
+
+    private boolean verifyFeedbackQuestion(FeedbackQuestion oldQuestion,
```

Comment:
Not fully related to the PR, probably a blocker to migration. Is `questionText` in the datastore entity missed out in the SQL entity? I believe it should be the data for this box here...

![image](https://github.com/TEAMMATES/teammates/assets/10182564/ac5ade30-c113-4998-bbd8-365908dc3f11)

Reply:
Alright I missed it out, it's in the details.